cpp/tests/copying/concatenate_tests.cpp

```diff
@@ -226,6 +197,53 @@ TEST_F(StringColumnTest, ConcatenateTooLarge)
   EXPECT_THROW(cudf::concatenate(input_cols), std::overflow_error);
 }
 
+TEST_F(StringColumnTest, ConcatenateLargeStrings)
+{
+  CUDF_TEST_ENABLE_LARGE_STRINGS();
```

Comment:
What's the long term plan for `CUDF_TEST_ENABLE_LARGE_STRINGS`? Will we need this forever or is it only until we turn on long strings by default?

Reply:
This will be temporary until long strings are turned on by default.
It will be useful until then in case specific tests require them.
Also, a future PR will introduce a new test fixture where this feature is turned on by default.
And then this test may be moved to use that fixture at that time.